src/modules/freia/test/ConfigTest.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2016 - 2020 European Spallation Source, ERIC. See LICENSE file
+//===----------------------------------------------------------------------===//
+///
+/// \file
+//===----------------------------------------------------------------------===//
+
+#include <freia/geometry/Config.h>
+#include <test/SaveBuffer.h>
+#include <test/TestBase.h>
+
+
+using namespace Freia;
+
+class ConfigTest : public TestBase {
+protected:
+  Config config;
+  void SetUp() override {}
+  void TearDown() override {}
+};
+
+TEST_F(ConfigTest, Constructor) {
+  ASSERT_EQ(config.NumPixels, 0);
+  ASSERT_EQ(config.NumCassettes, 0);
+}
+
+// Compare calculated maxpixels and number of fens against
+// ICD
+struct RingCfg {
+  uint8_t Ring;
+  uint16_t FENs;
+  uint16_t FENOffset;
+  uint16_t VMMOffset;
+};
+
+
+// This table generated from the ICD and will be
+// compared to the calculated values
+std::vector<RingCfg> FullConfig {
+  {  0, 2,  0,  0},
+  {  1, 2,  2,  8},
+
```

Comment:
Is there a reason for the blank line here?

Reply:
Yes that one is super important ;-)

No, not really. Must have slipped in by mistake.